Plantix/buyer-app/log-verification-utility/flow-2-resp.json

```diff
@@ -0,0 +1,108 @@
+{
+    "success": false,
+    "response": {
+        "message": "Logs were not verified successfully",
+        "report": {
+            "search_full_catalog_refresh": {
+                "schemaErr0": "/context/timestamp must match format \"rfc3339-date-time\"",
+                "timestamp_err": "Timestamp should be in date-time format"
+            },
+            "on_search_full_catalog_refresh": {
+                "prvdr0categories": "Support for variants is mandatory, categories must be present in bpp/providers[0]"
+            },
+            "select": {
+                "schemaErr0": "/context/timestamp must match format \"rfc3339-date-time\"",
+                "timestamp_err": "Timestamp not in RFC 3339 (YYYY-MM-DDTHH:MN:SS.MSSZ) Format",
+                "invalidItemId[F9nPDq0U58cOTPbOaWL1]": "Invalid item found in /select id: F9nPDq0U58cOTPbOaWL1",
+                "invalidItemId[YZ2SySvvAuPhIyec6gog]": "Invalid item found in /select id: YZ2SySvvAuPhIyec6gog",
+                "invalidItemId[4vkgqsSvRZEwywbpMeW2]": "Invalid item found in /select id: 4vkgqsSvRZEwywbpMeW2",
+                "providerId": "provider with provider.id: H6JplBH0HQxkVoKWAnD3 does not exist in on_search",
```

Comment:
Ok, i guess you need to do all steps in one go. So make sure that the on_search log from flow 1 include everything you use in flow-2...